### PR TITLE
chore: rename WCS to WC Shipping & Tax

### PR DESCRIPTION
**Pull request by frosso**
### Fixes 

Helps with https://github.com/Automattic/woocommerce-shipping-issues/issues/104
We would like to rename "WooCommerce Services" to "WooCommerce Shipping" and "WooCommerce Tax" (based on context, per https://a8c.slack.com/archives/CU6SYV31A/p1600282268248400 ). See this post for more info: https://wcship.wordpress.com/2020/06/29/research-spike-rename-woocommerce-service/

I renamed all I could find (excluding changelog entries).

### Accessibility

- [n/a] I've tested using only a keyboard (no mouse)
- [n/a] I've tested using a screen reader
- [n/a] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
- [n/a] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

### Detailed test instructions:

- Create store **without** WCS installed
- Set up store for order page prompt to install:
    a. WCS **not** installed
    b. U.S. Based
    c. No extra shipping extensions installed (e.g. 'UPS' live rates)
    d. Add shippable product
- Create order for shippable product
- Go to order edit page
- See shipping banner
Screen Shot 2020-09-16 at 1 29 33 PM
Other banners:

When setting up new store:
Screen Shot 2020-09-16 at 1 45 07 PM

On the Setup Wizard:
Screen Shot 2020-09-16 at 1 48 10 PM
Screen Shot 2020-09-16 at 1 48 51 PM

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 === WooCommerce Admin ===
-Contributors: woocommerce, automattic
+Contributors: automattic
```
Comment (timmyc) [inline readme.txt:2]: Looks like these changes might have slipped in from an old branch perhaps?

Comment (frosso) [reply readme.txt]: Thanks for catching that! It looks like I checked out the changes from the old `master`, rather than `main`. Fixed in 53cbfde9

**Review by timmyc — APPROVED** [review]
@frosso all is looking good for me here - I couldn't find anymore instances of "WooCommerce Services" in the code too. Made one minor note about still updating the actual docs on docs.woocommerce.com - but outside of that, all things are testing out well for me.

File: docs/features/onboarding.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ A few new WordPress options have been introduced to store information and settin
 * `woocommerce_task_list_welcome_modal_dismissed`. This option is used to show a congratulations modal during the transition between the profile wizard and task list.
 * `woocommerce_task_list_prompt_shown`. This option is used to conditionally show the "Is this card useful?" snackbar notice, shown once right after a user completes all the task list tasks.
 
-We also use existing options from WooCommerce Core or extensions like WooCommerce Services or Stripe. The list below may not be complete, as new tasks are introduced, but you can generally find usage of these by searching for the [getOptions selector](https://github.com/woocommerce/woocommerce-admin/search?q=getOptions&unscoped_q=getOptions).
+We also use existing options from WooCommerce Core or extensions like WooCommerce Shipping & Tax or Stripe. The list below may not be complete, as new tasks are introduced, but you can generally find usage of these by searching for the [getOptions selector](https://github.com/woocommerce/woocommerce-admin/search?q=getOptions&unscoped_q=getOptions).
```
[the remark this answers is not in the document]
Comment (frosso) [reply docs/features/onboarding.md:53]: Thanks for the note @timmyc ! I'll make sure to update them there as well, I'll either ask the WCDocs guild or update them myself 👍

**Review by c-shultz — APPROVED** [review]

**Pull request merged**